product_docs/docs/epas/12/epas_compat_spl/15_object_types_and_objects/02_object_type_components/01_object_type_specification_syntax.mdx

```diff
@@ -69,9 +69,11 @@ RETURN self AS RESULT
 ```
 
 !!! Note
-    -   The `OR REPLACE` option cannot be currently used to add, delete, or modify the attributes of an existing object type. Use the `DROP TYPE` command to first delete the existing object type. The `OR REPLACE` option can be used to add, delete, or modify the methods in an existing object type.
 
-    -   The PostgreSQL form of the `ALTER TYPE ALTER ATTRIBUTE` command can be used to change the data type of an attribute in an existing object type. However, the `ALTER TYPE` command cannot add or delete attributes in the object type.
+- The `OR REPLACE` option cannot be currently used to add, delete, or modify the attributes of an existing object type. Use the `DROP TYPE` command to first delete the existing object type. The `OR REPLACE` option can be used to add, delete, or modify the methods in an existing object type.
```

Comment:
@josh-heyer This is the problematic file you cited in <https://github.com/EnterpriseDB/docs/pull/1551#pullrequestreview-705682896>. Seems to work fine.